socialhome/streams/app/components/AuthorBar.vue

```diff
@@ -2,18 +2,18 @@
     <div>
         <div class="grid-item-author-bar mt-1">
             <div @click.stop.prevent="profileBoxTrigger" class="profilebox-trigger">
-                <img :src="author.imageUrlSmall" class="grid-item-author-bar-pic" />
-                {{ author.name }}
+                <img :src="author.image_url_small" class="grid-item-author-bar-pic" />
+                {{ author.name.length === 0 ? author.handle : author.name}}
```

Comment:
just `author.name.length ? ...` should be sufficient. Zero length is false. More Pythonic! :D